stdlib/LinearAlgebra/src/triangular.jl

```diff
@@ -272,6 +272,11 @@ Base.isstored(A::UpperTriangular, i::Int, j::Int) =
 @propagate_inbounds getindex(A::UpperTriangular, i::Integer, j::Integer) =
     i <= j ? A.data[i,j] : _zero(A.data,j,i)
 
+Base._reverse(A::UpperTriangular, dims::Integer) = (reverse((Matrix(A)); dims))
+Base._reverse(A::UpperTriangular, dims::Colon) = (reverse(Matrix(A)))
```

Comment:
This could return a Lower triangular based on the reversed data instead of a Matrix.